network/version.go

```diff
@@ -7,11 +7,16 @@ import "math"
 type Version uint
 
 const (
-	Version0 = Version(iota) // specs-actors v0.9.3
-	Version1                 // specs-actors v0.9.7
-	Version2                 // specs-actors v0.9.?
-	Version3                 // Coming soon
-	Version4                 // Who knows?
+	Version0 = Version(iota) // 00000000: genesis   (specs-actors v0.9.3)
+	Version1                 // 00041280: breeze    (specs-actors v0.9.7)
+	Version2                 // 00051000: smoke     (specs-actors v0.9.)
+	Version3                 // 00094000: ignition  (specs-actors v0.9)
+	Version4                 // 00128888: actors v2 (specs-actors v2.0.x (future))
+	Version5                 // 00148888: liftoff   (specs-actors v2.0.x (future))
 
+	// VersionCount is the number of versions defined.
+	VersionCount
```

Comment:
I omitted this intentionally because as soon as tried to use it I realised I'd set a trap for downstream things that might assume that `VersionCount` refers to a particular version, and break when new items were added here. Unless we major-version go-state-types every time we add a new one, I think this is unwise.

Reply:
I added this so I could per-allocate an array of versions, but I never ended up using it. No reason to keep it.